ci-operator/step-registry/upi/deprovision/heterogeneous/vpc/upi-deprovision-heterogeneous-vpc-ref.yaml

```
@@ -0,0 +1,27 @@
+ref:
+  as: upi-deprovision-heterogeneous-vpc
+  from: upi-installer
+  grace_period: 10m
+  commands: upi-deprovision-heterogeneous-vpc-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+  env:
+  - name: ADDITIONAL_WORKERS
+    default: "1"
```

```suggestion
    default: "2"
```

Pattern is to use 2 workers